src/core/operations_problem.jl

```diff
@@ -750,19 +750,16 @@ function _write_model_dual_results!(
             STORE_CONTAINER_DUALS,
             name,
             timestamp,
-            to_array(constraint),
+            constraint,
+            [name],
         )
 
         if exports !== nothing &&
            should_export_dual(exports[:exports], timestamp, problem_name_str, name)
             horizon = exports[:horizon]
             resolution = exports[:resolution]
             file_type = exports[:file_type]
-            df = axis_array_to_dataframe(constraint)
-            if names(df) == ["var"]
-                # Workaround for limitation in axis_array_to_dataframe.
-                DataFrames.rename!(df, [name])
-            end
+            df = axis_array_to_dataframe(constraint, [name])
```

Comment:
@jd-lara The resulting dataframes for duals will now have a relevant column name instead of `var`. I think this is an improvement but let me know if I got it wrong..

Reply:
I think this is fine but we should review the code in axis_array_to_df later too.